### PR TITLE
feature/REDBOX-224-mechanism-to-manage-accounts

### PR DESCRIPTION
**Pull request by gecBurton**
## Context

As a Product Owner I want control over who is signed up to the app via the admin.

## Changes proposed in this pull request

This change reverts https://github.com/i-dot-ai/redbox-copilot/pull/395

This:
* allows us to assign a superuser via a env var on deployment
* stops Cabinet Office users being able to up (but not in) in automatically

This does not include the OTP mechanism which is also needed.

Also the admin has been tidied up so that the user management is simpler 
image

## Relevant links

this forms part of: https://technologyprogramme.atlassian.net/browse/REDBOX-224

## Things to check

- [ ] I have added any new ENV vars in all deployed environments
- [ ] I have tested any code added or changed
- [ ] I have run integration tests

### CODE REVIEW

File: django_app/redbox_app/redbox_core/admin.py [hunk 1]
```diff
@@ -4,14 +4,25 @@
 
 
 class UserResource(admin.ModelAdmin):
-    exclude = ["password"]
+    fields = ["email", "is_superuser", "is_staff"]
```
Comment (gecBurton) [inline django_app/redbox_app/redbox_core/admin.py:7]: this is all we need to know for this app

File: django_app/redbox_app/redbox_core/admin.py [hunk 2]
```diff
@@ -4,14 +4,25 @@
...
 
 
 class FileResource(admin.ModelAdmin):
-    list_display = ["original_file_name", "user", "processing_status", "original_file", "core_file_uuid"]
+    list_display = ["original_file_name", "user", "processing_status"]
+
+
+
+class ChatMessageInline(admin.StackedInline):
```
Comment (gecBurton) [inline django_app/redbox_app/redbox_core/admin.py:15]: this is a little neater than putting the individual messages in their own admin

File: django_app/redbox_app/redbox_core/admin.py (hunk 1 above)
Comment (252afh) [reply django_app/redbox_app/redbox_core/admin.py:7]: I'd also suggest `last_login` so we can find stale admins

File: django_app/redbox_app/redbox_core/admin.py (hunk 2 above)
Comment (252afh) [reply django_app/redbox_app/redbox_core/admin.py:15]: We need to remember that any admins will need OS clearance when allowing any external admins

File: django_app/redbox_app/redbox_core/management/commands/create_admin_user.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+logger = logging.getLogger(__name__)
+logger.setLevel("DEBUG")
+
+
+class Command(BaseCommand):
+    help = """This should be run once per environment to set the initial superuser.
+    Thereafter the superuser should be able to login and assign new staff users via the admin.
+    """
+
+    def handle(self, *args, **kwargs):
```
Comment (252afh) [inline django_app/redbox_app/redbox_core/management/commands/create_admin_user.py:17]: Should we make this a migration and set the setting to non-nullable? e.g. make it a system setup command, so people can't find themselves locked out of their system

**Review by 252afh — APPROVED** [review]

File: django_app/redbox_app/redbox_core/admin.py (hunk 1 above)
Comment (gecBurton) [reply django_app/redbox_app/redbox_core/admin.py]: 9153bac0b660657a3738c9770582a49235b9571a

**Pull request merged**